README.md

```diff
@@ -74,6 +74,13 @@ Example user story: `As an admin I want to be able to create new users because I
 
 _These guidelines for creating user stories are an adaptation of [Simon's](https://github.com/simonlab) [**user-story-checklist**](https://github.com/SimonLab/user-story-checklist) repo. Check it out!_
 
+### Pull Requests
```

Comment:
@iteles I _think_ the [`style-guide`](https://github.com/dwyl/style-guide#pull-requests) should _briefly_ address the **commit messages** and _link_ to `contributing` (_this repo_) which should include _all_ the detail. Preferably with pictures/examples.